### DISCUSSION TITLE
Lyric issue

### DISCUSSION THREAD

**Discussion by Yumega** [TT-ReBORN/Georgia-ReBORN#5] [Q&A] [answered]
It can display the whole lyrics with white color, but the color of the lyric is singing is not yellow, it's hard to know which sentence is singing.

**Comment by Yumega** [comment, marked as answer]
how to solve

**Reply by TT-ReBORN** [reply]
Hi Yumega,

only synched lyrics are displayed in yellow color ( or in neon-color if using the neon themes ), these synched lyrics 
**IF THEY ARE AVAILABLE IN THE SOURCE** are downloaded as .lrc files in your lyrics folder. If there are no synched lyrics for the song, non-synched lyrics will be downloaded and displayed always with white text without the highlighted yellow color...

**Reply by Yumega** [reply]
Another question, the theme has to load each time start Foobar. is there a method to avoid the loading page?

**Reply by TT-ReBORN** [reply]
No, because it's loading all the needed Javascript files to work properly. It's the same like in the original Georgia theme from Mordred... So there is no way to skip the loading, also if you have a fast CPU it should be very fast...

**Reply by TT-ReBORN** [reply]
Also make sure you have foobar with the theme installed on a fast SSD drive and for the fastest performance have your music also stored on a SSD. Otherwise if your music is on a large HDD, you need to wait until your HDD is typically waking up from sleep mode to initialize your music...
